server/coma/schemaMatching.ts

```diff
@@ -11,19 +11,31 @@ const execAsync = promisify(exec);
  *
  * @param srcTables in format schema.tables
  * @param trgTables in format schema.tables
+ * @param thesaurus explanation: see ISchemaMatchingRequest.ts
  */
 export default async function getSchemaMatching(
   srcSql: string,
-  targetSql: string
+  targetSql: string,
+  thesaurus?: string
 ): Promise<Array<ISchemaMatchingResponse>> {
   await ensureDirExists("coma/schemas");
-  const srcFile = await createSchemaDump(srcSql);
-  const targetFile = await createSchemaDump(targetSql);
+  const srcFile = await createFile(srcSql);
+  const targetFile = await createFile(targetSql);
   await ensureDirExists("coma/results");
   const resultFile = "coma/results/" + randomUUID() + ".json";
-  await execAsync(
-    `java -cp "coma/*" de.wdilab.coma.integration.COMA_API ${srcFile} ${targetFile} ${resultFile}`
-  );
+  if (!thesaurus)
+    await execAsync(
+      `java -cp "coma/*" de.wdilab.coma.integration.COMA_API ${resultFile} ${srcFile} ${targetFile}`
+    );
+  else {
+    // synonyms don't seem to do much, so just put everything into abbrevs
+    const synonyms = await createFile("");
+    const abbrevs = await createFile(thesaurus);
```

Comment:
Dachte der Thesaurus wäre für Synonyme :) vllt hab ich das falsch verstanden...aber ist ja nicht so wichtig, solange die im Command in der richtigen Reihenfolge stehen..

Reply:
Jo, aber irgendwie hatten die nur einen Effekt, wenn ich sie dem Schema Matcher als Abkürzungen überrgebe...